### ISSUE TITLE
WA - Close the loop on invalid MSH-3 "ProPhase"

### ISSUE THREAD

**Issue by BerniXiongA6** [CDCgov/prime-reportstream#6569]
## Problem statement

WA DOH wants to know when ProPhase has sent correct data that previously had invalid MSH-3 errors so they can close the loop on this issue.

## What you need to know

- See [previous ticket # 5669](https://app.zenhub.com/workspaces/onboarding--operations-6166edbd409257001e09f1b2/issues/cdcgov/prime-reportstream/5669) for RS engineering work that was completed on this issue. 

- WA DOH knows we've previously asked ProPhase to send the data since the resubmissions timeframe has expired and we've also fixed the errors.

- New submitter for ProPhase (of 7/28/2022) is anthony.millan@healthcareintegrations.com.

## Acceptance criteria
Ticket can be closed when:
- [x] Sender contact has confirmed they've sent the data successfully
- [ ] WA DOH has been alerted of the successful submission

## To do

- [x] Follow up with Anthony and cc: @oslynn to confirm 
- [x] Follow up with WA DOH with status

**Comment by BerniXiongA6** [comment]
Per @oslynn: The organization.yml file shows that RS is already setup to translate messages for WA-PHD. Files are validated and is ready to translate to WA-PHD according to what WA-PHD wants. Sending email update to WA DOH with this update.

**Comment by BerniXiongA6** [comment]
Hey @jbiskie when you're back in, can we [discuss ticket # 5669](https://app.zenhub.com/workspaces/onboarding--operations-6166edbd409257001e09f1b2/issues/cdcgov/prime-reportstream/5669) which prompted this new ticket? Though we've had the ProPhase sending contact Anthony/HealthCare Integrations send two new messages, sounds like there were 155 more they're waiting on (I think this was captured in an email sent by Jenni to you before MPX)?

Per Jenni at WA DOH:

> Hi Berni,
> 
>  
> 
> We confirmed that we did receive two messages.  We are waiting for the remaining 115 resubmissions.  Please let us know when they have been transmitted so that we may validate and follow up.
> 
>  
> 
> Thank you,
> 
> Jenni

**Comment by jbiskie** [comment]
Emailed Anthony with the request and messages needed for resend.


[https://app.zenhub.com/files/304423150/f6dcca78-0db3-4ef7-a8b8-e0e9e906426a/download](https://app.zenhub.com/files/304423150/f6dcca78-0db3-4ef7-a8b8-e0e9e906426a/download
)

**Comment by BerniXiongA6** [comment]
@jbiskie sent file to Anthony at ProPhase with file containing 117 messages that need to be re-submitted to WA DOH.

> Hi Anthony,
> 
> We have a request from WA to have you resubmit some results from back around the time of Go Live. There was some uncertainty initially over them needing these again or not, and our retention period for the messages has since expired. Are you able to send the message IDs in the attached list to ReportStream again?
> 
> Thanks,
> 
> Joel Biskie (Flexion, Inc.)

cc: @oslynn

**Comment by jbiskie** [comment]
Anthony replied 9/7:

Corey reprocessed all that Joel had requested. Please double-check and let us know if you need anything else.


Emailed Jenni:

ProPhase confirmed that they resent the remaining submissions today. Can you please validate on your side?  (*)They just reported this completed, so may need to give it until the next batches come through.

**Comment by sliu1000** [comment]
Response from Jenni at WA State:

Thank you Joel.  Could you please send a filename or message control IDs for these resubmissions so we may confirm receipt?

Best,
Jenni

**Comment by sliu1000** [comment]
Update from WA State:

Happy Friday Joel.

We received resubmissions for the first 57 message control IDs on the list.  Thank you. 

Please let us know when the remainder are sent.

Best,
Jenni

**Comment by jbiskie** [comment]
Looked up all of the submissions, they appear to have gone to WA. Emailed Jenni with the filenames for each.

**Comment by shy2cdc** [comment]
The below is from Jenni 9/12 at 2:29pm

Thanks for troubleshooting with me Joel. 

When I searched our archive for files listed on your expanded set I found another file that we did not receive (cdcprime_testing_20220907071609_520183___neg.hl7)  marked in red text below.  I might be a coincidence, but I did notice that each file that failed to reach us was sent around 2AM or 2PM.  Does that time window have any significance for you?

 

cdcprime_testing_20220907011609_653145___neg.hl7      9/6/2022, 8:16:09 PM                             received

cdcprime_testing_20220907031612_996550___neg.hl7      9/6/2022, 10:16:12 PM                          received

`cdcprime_testing_20220907051604_756881___neg.hl7`      9/7/2022, 12:16:04 AM                          received

**``cdcprime_testing_20220907071609_520183___neg.hl7`      9/7/2022, 2:16:09 AM                             not received`**

cdcprime_testing_20220907091610_149282___neg.hl7      9/7/2022, 4:16:10 AM                             received

cdcprime_testing_20220907131613_565844___neg.hl7      9/7/2022, 8:16:13 AM                             received

cdcprime_testing_20220907151604_471799___neg.hl7      9/7/2022, 10:16:04 AM        received

cdcprime_testing_20220907171616_620781___neg.hl7      9/7/2022, 12:16:16 PM        received

cdcprime_testing_20220907191605_561311___neg.hl7      9/7/2022, 2:16:05 PM         not received

cdcprime_testing_20220907211618_739278___neg.hl7      9/7/2022, 4:16:19 PM               received

cdcprime_testing_20220907231607_031400___neg.hl7      9/7/2022, 6:16:07 PM               received

 

cdcprime_testing_20220907013513_520113.hl7                  9/6/2022, 8:35:13 PM              received

cdcprime_testing_20220907033509_198488.hl7                  9/6/2022, 10:35:09 PM            received

cdcprime_testing_20220907053526_385825.hl7                  9/7/2022, 12:35:26 AM            received

cdcprime_testing_20220907153552_931696.hl7                  9/7/2022, 10:35:52 AM            received

cdcprime_testing_20220907173524_561023.hl7                  9/7/2022, 12:35:24 PM            received

cdcprime_testing_20220907193544_283978.hl7                  9/7/2022, 2:35:44 PM    not received

cdcprime_testing_20220907213530_955730.hl7                  9/7/2022, 4:35:30 PM received

cdcprime_testing_20220907233547_399771.hl7                  9/7/2022, 6:35:47 PM received

**Comment by sliu1000** [comment]
Update from Joel to User:

Not that I'm aware of. The files we have logged on this side should be getting to you... What about on other days, do you have files during the 2:00 PM hour (which should be the 12:00 PM hour for you, I'm in Central). The morning one is more hit or miss because volume is going to be lower overnight and we may not have any for you then.

**Comment by jbiskie** [comment]
Joel,

I can confirm that we do regularly receive files from USDS at roughly two hour intervals from 4AM – 9PM PST. Our midday files tend to arrive around 12:45PM.  We regularly receive files at this time except for on 9/7/2022. 

Could we get these 3 transmission failures resubmitted?

cdcprime_testing_20220907071609_520183___neg.hl7      9/7/2022, 2:16:09 AM                   not received
cdcprime_testing_20220907191605_561311___neg.hl7      9/7/2022, 2:16:05 PM                       not received
cdcprime_testing_20220907193544_283978.hl7            9/7/2022, 2:35:44 PM         not received

Thank you,

Jenni

**Comment by sliu1000** [comment]
9/16/2022 - Update from User from WA State - Jenni:

I am still unable to locate these records.  Would it be possible to get them secure emailed?

Jenni

9/15/2022 - 3:40 PM - Biskie, Joel (CDC/DDPHSS/OD/HITSSU) (CTR)

Jenni,

I've resubmitted the 3 transmissions.

Thank you,
Joel

**Comment by BerniXiongA6** [comment]
OK to close per Jenni Risler via WA sync today 9/26/2022
